coffee-table-groups/coffee-table-groups.md

```diff
@@ -1,113 +1,116 @@
 
 ### Data Structures and Algorithms (DSA) Office Hours
 
-**When:**
+#### When:
 
-- Wednesdays at 4:00 PM ET
+- Wednesdays, 4:00-5:00 PM ET
 
-**Lead & Host:**
+#### Lead & Host:
 
 - Eddie Banner
 
-DSA Office Hours is an opportunity to come and ask questions about anywhere you get stuck on this week’s coding problems, or about this week’s topic. Find this week’s DSA topic and coding problems in #tech-interview-study-group. New topic and problems every week!
+DSA Office Hours is an opportunity to come and ask questions about anywhere you get stuck on this week’s coding problems, or about this week’s topic. Find this week’s DSA topic and coding problems in `#tech-interview-study-group`. New topic and problems every week!
 
-### Feelings Friday
+### Frontend Friday Folks Fighting CSSBattle.dev
 
-**When:**
+#### When:
 
-- Fridays at 8:00 PM ET
+- Fridays, 11:00 AM – NOON ET
 
-**Lead & Host:**
+#### Lead:
 
-- Ethan Freire
+- Jörn Bernhardt
 
-Feelings Friday was started at the Flatiron School coding bootcamp by its founders, Avi Flombaum & Adam Enbar, after noticing burnout among their first cohorts. Join a safe space to share what that's on your mind and hold space for others. It's a great way to unwind at the end of the week!
+#### Hosts:
 
-### Frontend Friday Folks Fighting CSSBattle.dev
+- Jörn Bernhardt
+- Micha Rodriguez
 
-**When:**
+On Fridays, we solve a new puzzle at [CSSBattle.dev](https://cssbattle.dev/). We meet in the `#co-working-room` for this Coffee Table Group, and can move our meeting to a breakout room so as not to disturb any co-working VC members. Feel free to join us and learn more about CSS! If you can't make it, be sure to play along on your own and post your answers in [Virtual Coffee's CSS Battles Discussion board](https://github.com/orgs/Virtual-Coffee/discussions/categories/css-battles)!
 
-- Every other Friday at 11:00 AM ET
+### Feelings Friday
 
-**Lead & Host:**
+#### When:
 
-- Jörn Bernhardt
+- Fridays, 8:00–9:00 PM ET
+
+#### Lead & Host:
+
+- Ethan Freire
 
-Every other Friday, we solve a new puzzle at [CSSBattle.dev](https://cssbattle.dev/). We meet in the #co-working-room for this Coffee Table, and can move our meeting to a breakout room so as not to disturb any co-working VC members. Feel free to join us and learn more about CSS! If you can't make it, be sure to play along on your own and post your answers in [Virtual Coffee's CSS Battles Discussion board](https://github.com/orgs/Virtual-Coffee/discussions/categories/css-battles)!
+Feelings Friday was started at the Flatiron School coding boot camp by its founders, Avi Flombaum & Adam Enbar, after they noticed burnout among their first cohorts. Join a supportive group where everyone has the opportunity to share what’s on their mind in a safe environment. It's a great way to unwind at the end of the week!
```

Comment:
**suggestion**: I think most of tech bootcamp goes with "bootcamp" rather than "boot camp". But feel free to drop this suggestion if you think "boot camp" is better.

```suggestion
Feelings Friday was started at the Flatiron School coding bootcamp by its founders, Avi Flombaum & Adam Enbar, after they noticed burnout among their first cohorts. Join a supportive group where everyone has the opportunity to share what’s on their mind in a safe environment. It's a great way to unwind at the end of the week!
```